Android/src/main/java/applicaster/analytics/firebase/FirebaseAgent.java

```diff
@@ -40,6 +45,8 @@
     public static final String GA_PREFIX = "ga_";
     public static final String SEND_USER_DATA = "Send_User_Data";
     public static final String USER_ID = "user_id";
+    public static final String SEND_BROADCAST_ACTION = "send_broadcast_from_rn";
```

Comment:
Please give it a more meaningful name.
What was the old SDK naming (missing interface method)?
Maybe we can use that :)

Reply:
Please be aware that in the following p.r: https://github.com/applicaster/zapp-login-plugin-emmys-cognito/pull/59/files
the key name is `userProperties`.

Reply:
You're confusing the action and the key...
The intent action is send_broadcast_from_rn (I'm registered to this action) and upon receiving an event, we have the key and properties that can be anything we want.
btw - 
val SEND_BROADCAST_ACTION = "send_broadcast_from_rn" is taken from here:
https://github.com/applicaster-plugins/react-native-broadcast-manager/pull/1/files - RNBroadcasterManager.kt

Reply:
Made another change - now I only call sendUserProperties when the EVENT_NAME  is "userProperties"